.audit-harness/scripts/gherkin-lint.sh

```
@@ -0,0 +1,111 @@
+done
+
+if [[ ! -d "$PATH_ARG" ]]; then
+  echo "gherkin-lint: path not found: $PATH_ARG" >&2
+  exit 2
+fi
+
+WARN_COUNT=0
+ERROR_COUNT=0
+
+warn() { echo "WARN  $1:$2 $3"; WARN_COUNT=$((WARN_COUNT + 1)); }
+err()  { echo "ERROR $1:$2 $3"; ERROR_COUNT=$((ERROR_COUNT + 1)); }
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)